backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/CsvApi.java

```diff
@@ -50,10 +52,47 @@ private static String discardMetadata(Request request, Response response) {
   }
 
   static String mergeMetadata(Request request, Response response) {
```

Comment:
would love to see a unit test for this one.
Could be copy of https://github.com/molgenis/molgenis-emx2/blob/b8b87bff8604937b6356f31709bf147ed21d96cf/backend/molgenis-emx2-webapi/src/test/java/org.molgenis.emx2.web/WebApiSmokeTests.java#L108 but then with a data file?